csharp/ql/src/semmle/code/csharp/dataflow/internal/FlowSummaryImpl.qll

```diff
@@ -643,6 +643,13 @@ module Private {
       )
     }
 
+    /**
+     * Holds if `spec` specifies summary component stack `stack`.
+     */
+    predicate interpretSpec(string spec, SummaryComponentStack stack) {
```

Comment:
The lines
```
          interpretSpec(inSpec, 0, input) and
          interpretSpec(outSpec, 0, output)
```
below might as well be updated to refer to this predicate. (I can't comment on those lines as they are to far from the diff.